### PR TITLE
[Issue #189] feat: add xec client

### PR DESCRIPTION
**Pull request by chedieck**
Description:
Turns `bchdService` into `grpcService` and makes it so that it accepts both XEC and BCH addresses. The environment variable `GRPC_NODE_URL` was replaced by `GRPC_BCH_NODE_URL` and `GRPC_XEC_NODE_URL`,
 

Test Plan:
1. Edit the `.env` file to include a BCH and XEC node, example:
```
GRPC_BCH_NODE_URL=bchd.fountainhead.cash:443  
GRPC_XEC_NODE_URL=grpc.fabien.cash:8335   
```
2. Run `make reset-dev`
3. After the containers are up, try:
```
curl http://localhost:3000/api/transactions/sync/qzqh7ej2vz26a9xaxq7capzfwgxt5gem9g8rvfxc5t
curl http://localhost:3000/api/transactions/sync/qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7quss2vxek
``` 
4. Now, after logging in to the dev user, go to http://localhost:3000/buttons and check that the  transactions are visible there.

Depends on:

- [x] #201

### CODE REVIEW

**Comment by Klakurka** [comment]
I can't get this to compile.

**Comment by chedieck** [comment]
> I can't get this to compile.

Can you post the errors you're getting?

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ https://paybutton.org
 
 ### Optional configuration
 - Enable _social login_ by filling up `.env` or `.env.local` file with your social provider credentials. You can get testing credentials and more detailed instructions [here](https://supertokens.com/docs/thirdpartyemailpassword/quick-setup/backend#2-initialise-supertokens).
-- Set `GRPC_NODE_URL` environment variable in `.env.local` or `.env` to a **BCHD** node to get access to its API.
+- Set `GRPC_BCH_NODE_URL` and `GRPC_XEC_NODE_URL` environment variables in `.env.local` or `.env` to a **BCHD** and **XECD** node, respectively, to get access to their API.
```
[the remark this answers is not in the document]
Comment (Klakurka) [reply README.md:24]: Makes sense to me to allow them to enter only one of them if they want to but we should probably make supporting that its own task.

**Review by Klakurka — CHANGES_REQUESTED** [review]
I'm getting this on the XEC call:

`{"statusCode":500,"message":"14 UNAVAILABLE: DNS resolution failed"}`

**Comment by chedieck** [comment]
> I'm getting this on the XEC call:
> 
> `{"statusCode":500,"message":"14 UNAVAILABLE: DNS resolution failed"}`

I am getting the same, though while trying to fix the bug I found out that _ocasionally_ it works. I think the problem is in the node, but I am still trying to confirm that.

**Comment by chedieck** [comment]
update: this now depends on #201. Edited description accordingly.

**Review by Klakurka — APPROVED** [review]

**Pull request merged**